File: Solutions/Java/src/test/java/KataTest.java
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class KataTest {
    private Kata fixture;

    @BeforeEach
    public void setUp() {
        fixture = new Kata();
    }

    @Test
    public void helloShouldReturnMessageWithSpecifiedName() {
        String name = "Ingage";
        assertEquals(String.format("Hello %s", name), fixture.hello(name));
    }
}

File: Solutions/Kotlin/src/test/kotlin/KataTest.kt
import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

class KataTest {
    private lateinit var fixture: Kata

    @BeforeEach
    fun setUp() {
        fixture = Kata()
    }

    @Test
    fun `hello should return message with specified name`() {
        val name = "Ingage"
        assertEquals("Hello $name", fixture.hello(name))
    }
}
